live-examples/js-examples/globalprops-encodeuri.html

```diff
@@ -0,0 +1,15 @@
+<pre>
+<code id="static-js">var uri = "https://mozilla.org/?x=шеллы";
+var encoded = encodeURI(uri);
+console.log(encoded);
+// expected output: https://mozilla.org/?x=%D1%88%D0%B5%D0%BB%D0%BB%D1%8B
+
+try {
+  console.log(decodeURI(encoded));
+} catch(e) { // catches a malformed URI
+  console.error(e);
+}
+
+// expected output: https://mozilla.org/?x=шеллы
```

Comment:
same comment about this line, I would have expected it at 9.

Reply:
Yup.